projects/openapi-cli/src/captures/streams/sources/har.ts

```diff
@@ -84,8 +85,15 @@ export class HarEntries {
 
       let requestBodyBuffer = interaction.request.body.buffer;
       let requestContentType = interaction.request.headers['content-type'];
-      let requestBodyEncoded =
-        requestContentType && requestBodyBuffer.toString('base64');
+
+      const requestBodyEncoded =
+        requestContentType &&
+        (await toBase64(
+          requestBodyBuffer,
+          (interaction.response.headers['content-encoding'] as
+            | string
+            | undefined) || undefined
```

Comment:
It's recommended to use `?? undefined` instead of `|| undefined` to avoid accidentally turning falsy but meaningful values such as `false` or `0` to `undefined` (doesn't make a difference here, this is more a general comment)